docs/guide/setup/alchemical_network_creation.rst

```diff
@@ -0,0 +1,61 @@
+.. _alchemical_network_creation:
+
+Alchemical Networks: Creation
+=============================
+
+In :ref:`the previous section <alchemical_network_model>` we detail the
+theory of :class:`.AlchemicalNetwork` objects and what they contain. Here
+we explain how you can go about creating a :class:`.AlchemicalNetwork`
+by combining its various components.
+
+Python API
+----------
+
+You can manually create a :class:`.AlchemicalNetwork` by creating a list
+of :class:`.Transformation` objects.
+The `cookbook on creating alchemical networks <../../cookbook/create_alchemical_network.html>`_
+demonstrates how to do this.
+
+Alchemical Network Planners
+---------------------------
+
+OpenFE provides convenience classes for creating :class:`.AlchemicalNetwork`.
+These currently include;
+
+* :class:`.RBFEAlchemicalNetworkPlanner`: creating relative binding free energy networks using :class:`.RelativeHybridTopologyProtocol`
+* :class:`.RHFEAlchemicalNetworkPlanner`: creating relative hydration free energy networks using :class:`.RelativeHybridTopologyProtocol`
+
+The `Relative Alchemical Network Planners cookbook <../../cookbook/rfe_alchemical_planners>`_
```

Comment:
PR for cookbook is here: https://github.com/OpenFreeEnergy/ExampleNotebooks/pull/124